.github/scripts/libjpeg-turbo/pack-tests.sh

```diff
@@ -6,8 +6,14 @@ LIBJPEG_TURBO_BUILD_PATH=$BUILD_PATH/libjpeg-turbo
 
 echo "::group::Pack libjpeg-turbo tests"
     mkdir -p $ARTIFACT_PATH
-    cd $LIBJPEG_TURBO_BUILD_PATH
-    zip -r $ARTIFACT_PATH/$TOOLCHAIN_NAME-libjpeg-turbo-tests.zip *
+
+    pushd $LIBJPEG_TURBO_BUILD_PATH
+        cp $TOOLCHAIN_PATH/$TARGET/bin/*.dll .
```

Comment:
I suggest to create a separate package with shared libraries and use it when it is needed for testing.

Reply:
DLLs moved to a separate artifact package.